server/monitor/credentials.go

```diff
@@ -0,0 +1,21 @@
+package monitor
+
+import (
+	"time"
+)
+
+func (m *Monitor) checkCredentials(force bool) {
+	if !force && (time.Now().Hour() != 0 || time.Now().Minute() != 0) {
+		return
+	}
+
+	credentials, err := m.client.GetAppCredentials(m.applicationID)
+	if err != nil {
+		m.api.LogDebug("Error getting client secret expire date", "error", err.Error())
```

Comment:
Let's log this as a warning if indeed the request is failing:
```suggestion
		m.api.LogWarn("Error getting client secret expire date", "error", err.Error())
```

Reply:
Done